testing/trino-testing/src/main/java/io/trino/testing/AbstractTestQueryFramework.java

```diff
@@ -190,7 +193,9 @@ public void ensureTestNamingConvention()
 
     protected Session getSession()
```

Comment:
maybe we rename this method to something like:
```
createQuerySession
```
so that it's explicit that it's not getter.

See for example `BaseJdbcConnectorTest#testTopNPushdownDisabled`:
```
        Session topNPushdownDisabled = Session.builder(getSession())
                .setCatalogSessionProperty(getSession().getCatalog().orElseThrow(), "topn_pushdown_enabled", "false")
                .build();
```
this getter is used twice there, but since each session is different, it should only be used once